### ISSUE TITLE
How to apply Google Closure Compiler for ng-bootstrap?

### ISSUE THREAD

**Issue by haiz** [ng-bootstrap/ng-bootstrap#1543]
I tried to apply Google Closure Compiler for ng-bootstrap but not success. 

Need your help, guys!

**Comment by pkozlowski-opensource** [comment]
You can check @mlaval repository that is comparing various build options (including Closure compiler) and is having ng-bootstrap as a dependency: https://github.com/mlaval/optimize-angular-app. 

I'm afraid that at this point we are not going to actively work on Closure compiler support but rather align with whatever angular-cli is doing (WebPack + UglifyJS as of today).

Having said the above we would be happy to do necessary changes in this library, if those are required to have Closure compiler working seamlessly. It is just we are not going to dig into the topic right now - if you are willing to investigate if changes are needed on the ng-bootstrap side and send PRs if so we would certainly review those and try to land.

In short: what you are asking for is kind of advanced and we would expect active involvement of people needing such advanced support.
